src/hooks/useToolApprovals.ts

```diff
@@ -60,7 +61,7 @@ export const useToolApprovals = (options: UseToolApprovalsOptions = {}) => {
       events.toolApprovalResponded({
         toolCallId,
         status,
-        approvedBy: currentUser.id,
+        approvedBy: currentUser.sub,
```

Comment:
I can't wait until events just use their validated `clientId` so we don't have to pass this through LiveStore events manually.